src/__tests__/parser.test.ts

```diff
@@ -0,0 +1,125 @@
+/* eslint-disable @typescript-eslint/no-unused-expressions */
+import { expect } from "chai";
+import { TokenMap, fetchTokenList } from "src/hooks/tokenList";
+import { TokenInfo } from "@uniswap/token-lists";
+import { parseCSV } from "src/parser";
+import { SafeInfo } from "@gnosis.pm/safe-apps-sdk";
+import BigNumber from "bignumber.js";
+
+let tokenList: TokenMap;
+let listedTokens: string[];
+let listedToken: TokenInfo;
+const unlistedTokenAddress: string =
+  "0x6b175474e89094c44da98b954eedeac495271d0f";
+
+let validReceiverAddress = "0x1000000000000000000000000000000000000000";
```

Comment:
We are initializing these now a few times throughout our tests. Maybe we can make a util file for this and just import them from there.

Reply:
I actually thought about the same. Some kind of test util to provide us with mockdata.

I'll put it together.

Reply:
Did this happen or will it come in a different PR?